pkgs/development/cuda-modules/generic-builders/multiplex.nix

```diff
@@ -3,7 +3,9 @@
   lib,
   cudaVersion,
   flags,
-  hostPlatform,
+  # NOTE: We would use backendStdenv to get the hostPlatform, but it's not available in the callPackage context
+  # we're using to call this file.
```

Comment:
This is about `cudaPackages.callPackage`?

Reply:
Correct -- this will be made more clear in the latest.

Reply:
There is a `backendStdenv` in `cudaPackages`, so its `callPackage` should be able to provision it

Reply:
We don't use `cudaPackages.callPackage` to call the multiplexed builder:

https://github.com/NixOS/nixpkgs/blob/3a45d48d581c59a4b7cdb272c08de9b408456763/pkgs/top-level/cuda-packages.nix#L90-L96

That's done mostly to avoid infinite recursion errors.

Reply:
Then this seems like we're institutionalizing a bug here

Reply:
I see that we were already doing this on master. Let's just shorten the comments:

- Instead of "not available in callPackage", "called by pkgs.callPackage"
- Clarify that this is an exceptional situation

Reply:
Resolved in 4c3d0c92e6329f2053d0c05c616672abee5caefc